lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -380,6 +399,29 @@ class ReportUIFeatures {
     this._document.body.removeChild(a);
     setTimeout(_ => URL.revokeObjectURL(href), 500);
   }
+
+  _updateStickyHeaderOnScroll() {
```

Comment:
does this need to be called once on startup?

Reply:
> does this need to be called once on startup?

I guess not since it starts out hidden

Reply:
TIL - https://stackoverflow.com/a/34095654
> The scroll event does not fire on every load, only when refreshing a page that was scrolled, or when navigating to an anchor directly.

So fresh view (top of page), state should be hidden, and the default is just that. Refresh after scrolling a bit, or click on a link w/ an anchor to the report, and a scroll event fires.